src/SDK/Metrics/Providers/GlobalMeterProvider.php

```diff
@@ -2,17 +2,19 @@
 
 declare(strict_types=1);
 
-namespace OpenTelemetry\Sdk\Metrics\Providers;
+namespace OpenTelemetry\SDK\Metrics\Providers;
 
-use OpenTelemetry\Metrics as API;
+use OpenTelemetry\API\Metrics as API;
 
 /**
  * https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/metrics/api.md#global-meter-provider
+ *
+ * @method static API\MeterInterface getMeter(string $name)
```

Comment:
This allows IDEs and such to be aware of the method even if it's not defined. Allowed me to remove the ignored error in the PHPStan config.